spec/omniauth-rails/railtie_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe OmniAuth::Rails::Railtie do
+  before do
+    OmniAuth::Rails::Railtie.initializers.each(&:run)
+  end
+
+  it 'should only allow POST requests' do
+    expect(OmniAuth.config.allowed_request_methods).to eq([:post])
+  end
+
+  it 'should only allow POST requests' do
```

Comment:
I don't think that this test name is correct.

Reply:
A product of late-night coding. Updated, thanks!